### DISCUSSION TITLE
The GUI doesn't appear

### DISCUSSION THREAD

**Discussion by Rich-Richie** [illsk1lls/ZipRipper#28] [Q&A]
After I double-click ZipRipper.cmd, cmd opened and it says "USE THE GUI TO PROCEED". However, there's no GUI appearing. I don't think I opened the wrong executable. What am I doing wrong?

(By the way, executing the cmd file triggered the User Account Control. This should be a portable software, so what's up with that?)

**Comment by illsk1lls** [comment]
ZipRipper uses admin rights. Some temp files created and moved between programdata\appdata can sometimes cause permission issues between sessions.

When you execute the script without zr-offline.txt present in the same folder you are in “Online Mode”, which downloads the necessary resources from the web.  You need to run in “Online Mode” at least once  to create zr-offline, you can do so by clicking the letters “JtR” in John’s hat.

Theres def an issue if you arent getting a GUI, what antivirus do you use and what are your system specs/os version
